catalog/spatial/csw/spatial-csw-source/pom.xml

```diff
@@ -207,22 +213,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.62</minimum>
+                                            <minimum>0.0</minimum>
```

Comment:
It wouldn't be a bad idea to move the test class for this base functionality back into this module. You could just create a concrete subclass in the test folder and use it to test against.

Then, the modules testing the concrete subclasses would only have to test the additional/changed functionality. Otherwise, you're relying on one particular concrete implementation to give you your code coverage.